composer.json

```diff
@@ -17,7 +17,8 @@
         "symfony/dependency-injection": "^2.8|^3.0",
         "symfony/routing": "^2.7|^3.0",
         "symfony/http-foundation": "^2.7|^3.0",
-        "league/oauth2-client": "^1.0"
+        "league/oauth2-client": "^1.0",
+        "stevenmaguire/oauth2-bitbucket": "^0.1.2"
```

Comment:
It's specific to the Bitbucket provider, we don't need to depend on it globally. If users want this feature - they will install it manually. See other providers